### ISSUE TITLE
convert tflite model quantized by per tensor, accuracy benchmark with sample app(labelimg), accuracy rate is much lower that quantized by per axies

### ISSUE THREAD

**Issue by joyb55933** [tensorflow/tensorflow#57745]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):ARMv8 mpu 
- TensorFlow installation (pip package or built from source): tensorflow 2.9.1

### 2. Code

Accuracy benchmark with labelimg, verify accuracy rate by Imagenet50k dataset(5k out of 50k),
quantzied model by per tensor (converter_experimental_disable_per_channel=True)

### 3. Benchmark
for inference time, can get about ~50% improvement (from ~6ms to ~3ms).
for accuracy rate, decrease a lot (from ~69%(per channel) to~10%(per tensor).

I wonder if there are some other convertor parameters that can be used for combination?
or is there any official benchmark data of per tensor models published to public for reference?

**Comment by mohantym** [comment]
Hi @joyb55933 !
Could you provide minimal stand alone code to replicate the behaviour. 
Thank you!

**Comment by joyb55933** [comment]
`model = tf.keras.Sequential([hub.KerasLayer('[https://tfhub.dev/google/imagenet/mobilenet_v1_100_224/classification/5'](https://tfhub.dev/google/imagenet/mobilenet_v1_100_224/classification/5%27), output_shape=[1001])])
model.build([None, 224, 224, 3])
converter = tf.lite.TFLiteConverter.from_keras_model(model)
converter.post_training_quantize=True
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.representative_dataset = representative_dataset
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
converter.inference_input_type = tf.int8
converter.inference_output_type = tf.int8
converter._experimental_disable_per_channel = True
quant = converter.convert()
open('test.tflite', 'wb').write(quant)`

Here is the convert and quantized minial code from tfhub keras model to tflite model.
After get the per-tensor model, i do accuracy benchmark with sample application "imagenet_image_classification_run_eval" under(https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/tools/evaluation/tasks)/imagenet_image_classification) as below:
`./imagenet_image_classification_run_eval --model_file=/run/media/mmcblk2p1/accuracy/model-ptq/mobilenet_v1_1.0_224_ptq_tflite_uint8_float32.tflite   --ground_truth_labels=ILSVRC2012_validation_ground_truth_2000.txt  --ground_truth_images_path=2k-jpeg/ --output_file_path=output --model_output_labels=labels.txt
`
but get much lower accuracy rate:
`Num evaluation runs: 2000
Preprocessing latency: avg=23756.3(us), std_dev=0(us)
Inference latency: avg=143510(us), std_dev=756(us)
Top-1 Accuracy: 0.0005
Top-2 Accuracy: 0.001
Top-3 Accuracy: 0.0015
Top-4 Accuracy: 0.002
Top-5 Accuracy: 0.003
Top-6 Accuracy: 0.003
Top-7 Accuracy: 0.004
Top-8 Accuracy: 0.0045
Top-9 Accuracy: 0.0055
Top-10 Accuracy: 0.0055
`

**Comment by mohantym** [comment]
Hi @sachinprasadhs !
Could you look at this issue.
Thank you!

**Comment by sachinprasadhs** [comment]
Above link which you have provided is broken, could you please confirm if the link which you are trying to check is https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/evaluation/tasks/imagenet_image_classification/README.md

**Comment by joyb55933** [comment]
> Above link which you have provided is broken, could you please confirm if the link which you are trying to check is https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/evaluation/tasks/imagenet_image_classification/README.md

yes, i follow this link to do accuracy benchmark

**Comment by pjpratik** [comment]
Hi @joyb55933 

The input images for the tensorflow hub models are expected to have color values in the range [0,1], following the [common image input](https://www.tensorflow.org/hub/common_signatures/images#input) conventions as per the [documentation](https://tfhub.dev/google/imagenet/mobilenet_v1_100_224/classification/5). 

The default normalization needs to be checked to be between [0,1] instead of [-1,1] which the tool does.

Thanks.
